arch/ppc64le-options.mk

```diff
@@ -8,5 +8,7 @@
 MACHINETYPE := pseries
 KERNELPARAMS :=
 MACHINEACCELERATORS :=
+CPUFEATURES := pmu=off
```

Comment:
cc @nitkon

Reply:
I will let "pmu=off" be the default option if no one objects. Because I noticed that current ppc64le supported pmu=off in nested virtualization. @nitkon